ansible/ansible.cfg

```diff
@@ -4,6 +4,7 @@ gathering           = explicit
 host_key_checking   = False
 inventory           = plugins/inventory/nodejs_yaml.py
 library             = plugins/library
+vars_plugins        = vars_plugins
```

Comment:
Can you move `load_secrets.py` to inside `plugins/vars`

Reply:
yeah, much better place for the plugin, I'll do it